cves/CVE-2013-2251.yml

```diff
@@ -174,8 +173,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: Should sanitize actions and redirects
```

Comment:
Clarify how this was done in the fix, or talk about how the input should be sanitized in this case

Reply:
I feel as though this goes beyond the specified `quick explanation of how it applies` asked for in the lessons description. If others disagree, please comment and I will add to the note. 

If personally curious as to how they fixed the vuln, it looks like they sanitized the action parameter input with a regex based character whitelist, then completely removed redirect prefixed parameters.